src/Components/Facility/ConsultationForm.tsx

```diff
@@ -200,22 +257,29 @@ export const ConsultationForm = (props: any) => {
         discharge_date: state.form.discharge_date,
         patient: patientId,
         facility: facilityId,
-        referred_to: state.form.suggestion === 'R' ? state.form.referred_to : undefined,
+        referred_to:
+          state.form.suggestion === "R" ? state.form.referred_to : undefined,
+        consultation_notes:
```

Comment:
Codacy found an issue: [Identifier 'consultation_notes' is not in camel case.](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5811785)